docs/articles/documentation/guides/concepts/typescript-and-coffeescript.md

```diff
@@ -42,6 +42,62 @@ Whenever TestCafe encounters TypeScript compilation errors, it includes correspo
 > and [selector.addCustomMethods](../../reference/test-api/selector/addcustommethods.md)
 > sections to learn how to extend selectors in TypeScript.
 
+#### Type Cast Page Elements
+
+TypeScript compilers can not automatically identify TestCafe objects that refer to DOM elements. Perform manual [type assertions](https://www.typescriptlang.org/docs/handbook/2/everyday-types.html#type-assertions) in your [client-side code](../basic-guides/obtain-client-side-info.md) to ensure correct TypeScript compilation.
+
+Specify the [HTMLElement](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement) data type to access the DOM element's generic HTMLElement interface.
+
+A [client function](../../reference/test-api/clientfunction/README.md) in the example below calls the [Element.scrollIntoView()](https://developer.mozilla.org/en-US/docs/Web/API/Element/scrollIntoView) method to scroll an element into view.
+
+```ts
+import { ClientFunction, Selector } from 'testcafe';
+
+const scrollIntoView = ClientFunction( (selector: Selector) => {
+    const element = selector() as unknown as HTMLElement;
+    element.scrollIntoView();
+});
+
+fixture`HTMLElement`
+    .page('https://example.com');
+
+test('Scroll element into view', async t => {
+    const bottomOfPage = Selector('#bottom-div');
+
+    await scrollIntoView(bottomOfPage);
+});
+```
+
+Pick element-specific data types, such as [HTMLOListElement](https://developer.mozilla.org/en-US/docs/Web/API/HTMLOListElement)), whenever appropriate.
```

Comment:
whenever appropriate 
is that part necessary

Reply:
Doesn't really explain much. Does this work:

```suggestion
To avoid compilation errors, pick element-specific data types, such as [HTMLOListElement](https://developer.mozilla.org/en-US/docs/Web/API/HTMLOListElement)).
```